cmd/client.go

```diff
@@ -330,6 +336,10 @@ func (t *promptingAPIInitializer) Initialize(store *store.Store) (models.User, e
 		return user, err
 	}
 
+	if !t.prompter.IsTerminal() {
+		return models.User{}, ErrorNoAPICredentialsAvailable
+	}
```

Comment:
Let's get some test coverage on this. We already have an interface and a mock in place, so hopefully it's trivial:

https://github.com/smartcontractkit/chainlink/blob/master/cmd/client_test.go#L147-L148
```golang
	mock := &cltest.MockCountingPrompter{}
	tai := cmd.NewPromptingAPIInitializer(mock)
```